sklego/feature_selection/mrmr.py

```diff
@@ -0,0 +1,216 @@
+import warnings
+
+import numpy as np
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import f_classif, f_regression
+from sklearn.feature_selection._base import SelectorMixin
+from sklearn.utils.validation import check_is_fitted, check_X_y
+
+
+def _redundancy_pearson(X, selected, left):
+    """Redundancy function for the MRMR feature selector algorithm
+
+    Parameters
+    ----------
+    X : array-like, shape=(n_samples, n_features,)
+        Training data. Used to compute redundancy of the training features.
+    selected : array-like.
+        List of indexes of the selected features at iteration i-th.
+    left : array-like.
+        List of indexes of the left features at iteration i-th. Mrmr will select a feature
+        from this list.
+
+    Returns
+    -------
+    np.ndarray, shape = (len(left), )
+        The array containing the redundancy score using pearson correlation.
+    """
+    # if len(selected) == 0:
+    #     return np.ones(len(left))
```

Comment:
I personally would keep this edge case even though in the code below can never happen

Reply:
This will prevent the code the be tested 100% because it's never hit.
What do you think?

Reply:
We are not aiming to achieve 100% coverage necessarily.
If you want to keep the edge cases _and_ the coverage, you may add a unit test specific for this function